node/nanoexpress/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "^0.12.6"
```

Comment:
can you remove caret ? there is @dependabot that take care of update (`PR` me to update to prevent breaking)

Reply:
Remove `^` symbol?

Reply:
I did it :)